### ISSUE TITLE
Streamline dedup process as part of the photo scan

### ISSUE THREAD

**Issue by sefininio** [LibrePhotos/librephotos#1158]
As described in the [Discord thread message](https://discord.com/channels/784619049208250388/1197856724011470918/1205384343581757470), the dedup via django admin fails for very large photo collections. So, a more robust solution is required:

Option A:
- Add another button in the UI that causes a scan like process, that runs over all photos and dedupes faces
Option B:
- Have the dedup operation part of the scan (so when scanning a photo, the last step is the dedup)
- Optionally, extend the above and add a checkbox that defaults to off - and only if it is checked, the scan will do the dedup as well (kinda opt-in and not the default)

**Comment by derneuere** [comment]
I wrapped the Django admin task in an AsyncTask, which means that it should work now even with large collections. Would be great if you could test it :)

**Comment by sefininio** [comment]
Hi,
Sorry for the late reply...

I tried on `2024w14p1` but no change... Is it merged?

**Comment by sefininio** [comment]
Hey @derneuere , testing on `2024w14p1` made no difference. Clicking on "Select all 91465 photos" does nothing

**Comment by derneuere** [comment]
Yes, it was merged, but apparently it does not work yet. Will have to create more tests for it, but do not have the time for that atm :)

**Comment by derneuere** [comment]
The defect discussed in this thread (the admin Deduplicate faces action doing nothing on large libraries) is fixed by #2020. The original request to run dedup as part of the scan or from the web UI is tracked in #2030.
